eventkit_cloud/utils/auth_requests.py

```diff
@@ -87,6 +87,8 @@ def get_cred(slug=None, url=None, params=None):
         env_slug = slug.replace('-', '_')
         cred = os.getenv(env_slug + "_CRED") or os.getenv(env_slug.upper() + "_CRED")
     if cred is not None and ":" in cred and all(cred.split(":")):
+        cred = cred.replace('@', '%40')
```

Comment:
Does it make sense just to urlencode the cred instead of picking specific special characters?